lib/gtfs_parser.rb

```diff
@@ -188,9 +204,9 @@ def parse_time(time)
     Time.local date.year, date.month, date.day, hour, minute
   end
 
-  # Checks to see whether a time object falls within n hours
-  def time_within?(hours, time)
-    compare_time = Time.now + (60 * 60 * hours)
+  # Checks to see whether a time object falls within n minutes
+  def time_within?(minutes, time)
```

Comment:
Good, but I don't see any changes to `DEPARTURE_FUTURE_HOURS`. Doesn't this mean that bojangles only looks for departures within three minutes now?